README.md

```diff
@@ -1,13 +1,43 @@
-# alphatraining
-new repository to pratice on devops assets
-Training path to start this journey will be the following one: 
-Git
-Visual Studio
-Docker
-Kubernetes
-Terraform
-Helm
-Ansible
-Packer
-S3
-Artifactory
+# 1. Alphatraining
+
+<!-- TOC -->autoauto- [1. Alphatraining](#1-alphatraining)auto    - [1.1. Git](#11-git)auto    - [1.2. Visual Studio Code](#12-visual-studio-code)autoauto<!-- /TOC -->
```

Comment:
auto tags should not be here. Update usersettings as described here : 
https://github.com/AlanWalk/markdown-toc/issues/65